lib/sass_builder.dart

```diff
@@ -33,9 +33,13 @@ class SassBuilder implements Builder {
 
   final _log = new Logger('sass_builder');
   final String _outputExtension;
+  final String _outputStyle;
```

Comment:
Where are the different styles defined?

Reply:
@kevmoo https://github.com/sass/dart-sass/blob/master/lib/src/visitor/serialize.dart#L1133-L1157